axelor-account/src/main/resources/views/InvoicePayment.xml

```diff
@@ -61,9 +61,9 @@
     <panel name="discountPanel" title="Financial discount" showIf="applyFinancialDiscount"
       if="__config__.app.getApp('account')?.getManageFinancialDiscount()">
       <field name="applyFinancialDiscount" readonly="true"/>
-      <field name="$totalAmountWithFinancialDiscount" type="decimal"
+      <field name="totalAmountWithFinancialDiscount" type="decimal"
```

Comment:
Please remove the `type` attribute since this is not a dummy field anymore

Reply:
Thanks it's corrected